platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -75,6 +75,14 @@ This change impacts:
 - Project selection
 - Configuration selection in dependency report tasks
 
+==== Swift 6 Support
```

Comment:
💅 I guess it's not about Swift 6. Consider changing the title to something like "Swift SDK Selection"